helm/micro-integrator/plans/medium.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resources:
+  requests:
```

Comment:
@ThilinaManamgoda just to clarify, are these the recommended, minimum resource values for Micro Integrator?

Or are these values simply based on the Pivotal based Plan concept (where a higher Plan i.e. `medium` > `small`, requests for a significant, min amount of resources, depending on the plan type)?